src/components/Button/PlaceFavoriteButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from "react"
+
+import useTrackContext from "decentraland-gatsby/dist/context/Track/useTrackContext"
+import { Button } from "decentraland-ui/dist/components/Button/Button"
+import Icon from "semantic-ui-react/dist/commonjs/elements/Icon"
+
+import { AggregatePlaceAttributes } from "../../entities/Place/types"
+
+import "./PlaceFavoriteButton.css"
+
+export type PlaceFavoriteButtonProps = {
+  place: AggregatePlaceAttributes
+  onClick?: (
+    e: React.MouseEvent<MouseEvent>,
+    place: AggregatePlaceAttributes
+  ) => void
+}
+
+export default function PlaceFavoriteButton({
```

Comment:
- user `React.memo` when possible
- this component requires a loading state for itself otherwise user won't know if action is taking place